### PR TITLE
Ingress rules for ms-pileup.

### PR DESCRIPTION
**Pull request by arooshap**
Hi @amaltaro, could you please review this?

### CODE REVIEW

File: kubernetes/cmsweb/ingress/ing-dmwm.yaml [hunk 1]
```diff
@@ -80,3 +80,10 @@ spec:
             name: ms-unmer-t2t3
             port:
               number: 8242
+      - path: /ms-pileup/data
```
Comment (amaltaro) [inline kubernetes/cmsweb/ingress/ing-dmwm.yaml:83]: should we simply use the same pattern used for the other WMCore services (different than MSUnmerged)? Thus, defining this path is `/ms-pileup` only.

**Review by amaltaro — COMMENTED** [review]
Aroosha, these changes look good to me. Still, I left a comment along the code for your consideration. Thanks for providing this.

**Comment by muhammadimranfarooqi** [comment]
@amaltaro 

I just noticed that port `8249` is also used by `reqmon`. 

```
[mimran@lxplus8s12]~/ k get svc -n dmwm | grep 8249
ms-pileup         ClusterIP   10.254.129.132   <none>        8249/TCP,18249/TCP   9d
reqmon            ClusterIP   10.254.200.96    <none>        8249/TCP,18249/TCP   206d
```

For consistency, I suggest to use different port for this service.

File: kubernetes/cmsweb/ingress/ing-dmwm.yaml (hunk 1 above)
Comment (arooshap) [reply kubernetes/cmsweb/ingress/ing-dmwm.yaml]: Hi @amaltaro, thanks for the review. I have updated the path.

**Comment by amaltaro** [comment]
@muhammadimranfarooqi Hi Imran, sorry for missing your question. You are correct, sorry about this.
I am going to make a WMCore ticket and ask Valentin to follow this up. We will have to update this service which is already running in dev/testbed cluster.

**Comment by vkuznet** [comment]
@arooshap , please change port to 8241, I already made necessary other changes, please see https://github.com/dmwm/deployment/pull/1251 (and corresponding links in it).

**Comment by vkuznet** [comment]
+1 from my side.

**Comment by arooshap** [comment]
Hi @vkuznet,
I have made the relevant changes here. I have merged your PR in the deployment repository, and I am building the new frontend image. I will update once the relevant changes are deployed in the testbed.

**Comment by amaltaro** [comment]
These changes look good to me (and now with the correct port). Thank you!

**Pull request merged**